src/bundle/Resources/config/services/events.yaml

```
@@ -3,6 +3,8 @@ services:
         autowire: true
         autoconfigure: true
         public: false
+        bind:
+            $logLevel: 'error'
```

That is a kinda risky bind. We may want different services to use different errors in different circumstances. It would be better to inject it directly into the relevant service.

Or, in lieu of that and in light of another one of my comments:
```suggestion
            string $logLevel: 'error'
```